### ISSUE TITLE
Can I use comma as separator of thousands on axis numbers?

### ISSUE THREAD

**Issue by wenlibin02** [veusz/veusz#306]
Hi, 

I am looking a way to change the display format of the numbers on the axis. The required format is to use comma as separator for thousands. For example: 1200 should be formatted as "1,200". 

Any idea?

Bests,
Libin

**Comment by jeremysanders** [comment]
No. You can use engineering suffixes however, with the "%VE" formatting option. There's also the "%Ve" option which gives scientific notation.

**Comment by fabricix** [comment]
I see all formats use comma to separate decimal. The program have this convention too ( for example in label offset 1**,**5pt). 
- Can I change the default decimal separator in the program to _dot_ or _comma_)? 
- The SI (International System of Units) supports both symbols, so could this be a relevant option to include in the user preferences?
